CleverRaven/Cataclysm-DDA
# [WIP] Option to use Grass Sheets for Door Curtains

#### Summary
Content "Door Curtains from grass sheets"


#### Purpose of change
Mainly due to portal storms, it is important to be able to build doors to lock yourself indoors. If you do not have access to metalworking and/or cotton, this can be quite difficult. Allowing the player to use grass sheets instead of sheets should fix this.
Fixes #55789


#### Describe the solution
A new type of door that is like the curtain door but made from grass sheets. Deconstructs to grass sheets. Smashing might yield grass yarn. Can also be opened and closed by the portal storm opening and closing of doors.


#### Describe alternatives you've considered
The construct cost of the old sheet door seems a little strange when it comes to the string and the string substitutes. I considered to update it to match the grass sheet curtain recipe, but decided against it.


#### Testing
TBD


#### Additional context
TODO:
- Test it
- check spelling and grammar
- Make sure I didn't miss to update any IDs
